### PR TITLE
helm-ls: init at 0.0.3

### PR DESCRIPTION
**Pull request by stehessel**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by ocfox** [comment]
Result of `nixpkgs-review pr 227150` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>helm-ls</li>
  </ul>
</details>

**Review by ocfox — APPROVED** [review]
LGTM, and work fine on aarch64-darwin.

**Comment by felschr** [comment]
Result of `nixpkgs-review pr 227150` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>helm-ls</li>
  </ul>
</details>

**Review by felschr — CHANGES_REQUESTED** [review]

**Review by felschr — APPROVED** [review]

File: pkgs/development/tools/misc/helm-ls/default.nix [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+, testers
+, helm-ls
+}:
+buildGoModule rec {
```
Comment (natsukium) [inline pkgs/development/tools/misc/helm-ls/default.nix:8]:
```suggestion
}:

buildGoModule rec {
```

**Comment by natsukium** [comment]
`language-server` are managed under this directory.
https://github.com/NixOS/nixpkgs/tree/master/pkgs/development/tools/language-servers

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -1554,6 +1554,8 @@ with pkgs;
 
   headset-charge-indicator = callPackage ../tools/audio/headset-charge-indicator { };
 
+  helm-ls = callPackage ../development/tools/language-servers/helm-ls { };
```
Comment (natsukium) [inline pkgs/top-level/all-packages.nix:1557]: There is a language servers section around line 17300.

**Review by felschr — APPROVED** [review]

**Review by NickCao — APPROVED** [review]

File: pkgs/development/tools/language-servers/helm-ls/default.nix [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=${version}"
+  ];
+
+  postInstall = ''
+    mv $out/bin/helm-ls $out/bin/helm_ls
+    installShellCompletion --cmd roxctl \
```
Comment (SuperSandro2000) [inline pkgs/development/tools/language-servers/helm-ls/default.nix:32]: roxctl?

Comment (stehessel) [reply pkgs/development/tools/language-servers/helm-ls/default.nix:32]: Ah crap, copy&pasted... Thanks for the catch, I'll create a fix.

Comment (stehessel) [reply pkgs/development/tools/language-servers/helm-ls/default.nix:32]: fix: https://github.com/NixOS/nixpkgs/pull/227689

**Pull request merged**